src/screens/Customers/components/CustomerCard.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import { Card, CardActions, Typography, IconButton } from '@material-ui/core';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    arrow: {
+      color: theme.palette.text.secondary,
+      fontSize: '15px',
```

Comment:
Can we just remove this and for the `ArrowForwardIosIcon` put `size='small'`?

https://material-ui.com/api/icon/